next-i18next.config.js

```diff
@@ -2,5 +2,8 @@ module.exports = {
   i18n: {
     defaultLocale: 'en',
     locales: ['en', 'es'],
+    react: {
+      useSuspense: false,
+    }
```

Comment:
Can't use both Suspense (which we don't use) and i18next. Sometimes the app gets confused unless we explicitly tell it we're not using it.

Reply:
Consider inline comment. ✍️

Reply:
```suggestion
    react: {
      useSuspense: false, //Explicitly set this to false to avoid conflicts with our i18n system
    }
```